python/lsst/summit/extras/plotting/psfPlotting.py

```diff
@@ -473,6 +503,151 @@ def plotData(
     axs[1, 2].axvline(eQuartiles[2], color="grey", lw=1)
 
 
+def plot_triangle(ax, center=(0, 0), size=1.0, angle_deg=0.0, color="blue"):
```

Comment:
Please don't introduce a mix of camelCase and snake_case - the rest of this package is camel (and so is one of the other functions you're adding). Could you change the functions and variables to be consistent everywhere please?

Reply:
This function is now dropped. One of the new functions added has an optional variable that is snake case, since that's the name used in `treegp` (which follows snake case). Would you prefer consistency between packages, or consistency within a file here (at the expense of the former)?